keps/sig-node/2254-cgroup-v2/README.md

```diff
@@ -88,13 +88,13 @@ supported on cgroup v2.
 
 - CRI-O+crun: support cgroups v2
 
-- runc support for cgroups v2 is work in progress [current status](#current-cgroups-usage-and-the-equivalent-in-cgroups-v2)
+- runc: support cgroup v2 since [v1.0.0-rc93](https://github.com/opencontainers/runc/releases/tag/v1.0.0-rc93)
```

Comment:
Since rc91 experimentally. Graduated from experimental in rc93.

Reply:
Done, thanks.